src/saving_tools.jl

```diff
@@ -329,16 +338,18 @@ istaggable(x) = x isa AbstractDict
 
 
 """
-    struct2dict(s) -> d
+    struct2dict(::Type{DT},s) -> d where {DT<: AbstractDict}
```

Comment:
```suggestion
    struct2dict([type = Dict, ] s) -> d
```

Reply:
I went ahead and made this change, but just out of curiosity, does the notation with the brackets imply just another method(as opposed to obviously kwargs options at the end of a function). I understood what it does, but wasn't sure when to use that type of notation.

Reply:
the bracket notation inside documentation strings means optional _positional_ arguments. Of course for Julia this also coincides with different methods.